modern/src/common/components/DelayedTooltip.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import Tooltip from '@mui/material/Tooltip';
+
+const DelayedTooltip = ({
```

Comment:
This is not what I asked. I'm talking about something like this:

https://github.com/traccar/traccar-web/blob/master/modern/src/common/theme/components.js